@here/harp-mapview/lib/Utils.ts

```diff
@@ -1624,7 +1624,12 @@ export namespace MapViewUtils {
         objectSize: MemoryUsage,
         visitedObjects: Map<string, boolean>
     ): void {
-        if (texture === null || texture === undefined || texture.image === undefined) {
+        if (
+            texture === null ||
+            texture === undefined ||
+            texture.image === undefined ||
+            texture.image === null
+        ) {
```

Comment:
I think you can simplify this test a bit with something like:
   if (!texture?.image) return

Reply:
I thought about that, but remembered something about the coding guidelines to not use truthy/falsy values.. not sure we still comply with that?